denylist/src/settings.rs

```diff
@@ -36,7 +40,7 @@ impl Settings {
     /// Environemnt overrides have the same name as the entries in the settings
     /// file in uppercase and prefixed with "DENYLIST_". For example
     /// "DENYLIST_LOG" will override the log setting.
-    pub fn new<P: AsRef<Path>>(path: Option<P>) -> Result<Self> {
+    pub fn new<P: AsRef<Path>>(path: Option<P>) -> Result<Self, config::ConfigError> {
```

Comment:
Why did this change? Seems unneeded? If you want to include this change you should also remove config::ConfigError from this crate's  Error definition?

Reply:
Reverted in dbf8dd7